splitter.config.js

```diff
@@ -0,0 +1,20 @@
+var path = require('path');
+
+function resolve(filePath) {
+  return path.resolve(__dirname, filePath);
+}
+
+var isProduction = process.argv.indexOf("-w") === -1;
+console.log("Compiling for " + (isProduction ? "production" : "development") + "...");
+
+module.exports = {
+    entry: resolve("src/Nightwatch.fsproj"),
+    outDir: resolve("out"),
+    babel: {
+    //   presets: [["es2015", { modules: false }]],
+    //   sourceMaps: true,
```

Comment:
You can uncomment the second lines to get the source maps pointing to the F# code, but I thought it may be interesting to debug the generated JS code. The first line can remain commented as it's not necessary to compile ES2015 JS code (RN can understand it) and it's much more readable when debugging.